assignments/lambda-classes.js

```diff
@@ -1 +1,193 @@
 // CODE here for your Lambda Classes
+class Person {    // Parent Class
+  constructor(info) {
+    this.name = info.name;
+    this.age = info.age;
+    this.location = info.location;
+  }
+  speak() {
+    console.log(`Hello, my name is ${this.name}, I am from ${this.location}`);
+  }
+}
+
+class Instructor extends Person {    // Child Class
+  constructor(instructorInfo) {
```

Comment:
If you ever want to you can use a generic naming convention for supers/constructors across the board (something like attr or props). It can make keeping everything consistent a whole lot easier.